test/smtp.test.lua

```diff
@@ -146,6 +168,48 @@ test:test("smtp.client", function(test)
                   m.text,
                   "Subject: =%?utf%-8%?b%?YWJjZGVmZ2hpamvRj2xtbm9wcXJzdHV2d3h5eg==%?=", ""))
     test:is(subj, 1, 'subject codes >127')
+
+    r = client:request(addr, '3xx@tarantool.org',
+            'receiver@tarantool.org',
+            'mail.body')
+    test:is(r.reason, 'MAIL failed: 354 (SMTP error)', 'Errors 3xx')
+
+    r = client:request(addr, '4xx@tarantool.org',
+            'receiver@tarantool.org',
+            'mail.body')
+    test:is(r.reason, 'MAIL failed: 451 (SMTP error)', 'service unavailable')
+
+    r = client:request(addr, '5xx@tarantool.org',
+            'receiver@tarantool.org',
+            'mail.body')
+    test:is(r.reason, 'MAIL failed: 510 (SMTP error)', 'unexisting recipient')
+
+    r = client:request(addr, '10xx@tarantool.org',
+            'receiver@tarantool.org',
+            'mail.body')
+    test:is(r.reason, 'response reading failed (SMTP error)', 'unexisting recipient')
+
+    r = client:request(addr, 'sender@tarantool.org',
+            '3xx@tarantool.org',
+            'mail.body')
+    test:is(r.reason, 'RCPT failed: 354 (SMTP error)', 'Errors 3xx')
+
+    r = client:request(addr, 'sender@tarantool.org',
+            '4xx@tarantool.org',
+            'mail.body')
+    test:is(r.reason, 'RCPT failed: 421 (SMTP error)', 'service unavailable')
+
+    r = client:request(addr, 'sender@tarantool.org',
+            '5xx@tarantool.org',
+            'mail.body')
+    test:is(r.reason, 'RCPT failed: 510 (SMTP error)', 'unexisting recipient')
+
+    r = client:request(addr, 'sender@tarantool.org',
+            '10xx@tarantool.org',
+            'mail.body')
```

Comment:
Could you indent arguments on newlines?
```lua
    r = client:request(addr, 'sender@tarantool.org',
                       '5xx@tarantool.org',
                       'mail.body')
    test:is(r.reason, 'RCPT failed: 510 (SMTP error)', 'unexisting recipient')
```

Reply:
Done.

Reply:
Sorry, but indentation is still not perfect.

![Screenshot from 2022-01-11 09-44-37](https://user-images.githubusercontent.com/1151557/148894248-7699d00c-3353-4507-965b-fab9217099ec.png)

I have fixed it in a separate patch and put it on top of your branch:

```diff
--- a/test/smtp.test.lua
+++ b/test/smtp.test.lua
@@ -170,43 +170,43 @@ test:test("smtp.client", function(test)
     test:is(subj, 1, 'subject codes >127')
 
     r = client:request(addr, '3xx@tarantool.org',
-                        'receiver@tarantool.org',
-                        'mail.body')
+                       'receiver@tarantool.org',
+                       'mail.body')
     test:is(r.reason, 'MAIL failed: 354 (SMTP error)', 'Errors 3xx')
 
     r = client:request(addr, '4xx@tarantool.org',
-                        'receiver@tarantool.org',
-                        'mail.body')
+                       'receiver@tarantool.org',
+                       'mail.body')
     test:is(r.reason, 'MAIL failed: 451 (SMTP error)', 'service unavailable')
 
     r = client:request(addr, '5xx@tarantool.org',
-                        'receiver@tarantool.org',
-                        'mail.body')
+                       'receiver@tarantool.org',
+                       'mail.body')
     test:is(r.reason, 'MAIL failed: 510 (SMTP error)', 'unexisting recipient')
 
     r = client:request(addr, '10xx@tarantool.org',
-                        'receiver@tarantool.org',
-                        'mail.body')
+                       'receiver@tarantool.org',
+                       'mail.body')
     test:is(r.reason, 'response reading failed (SMTP error)', 'unexisting recipient')
 
     r = client:request(addr, 'sender@tarantool.org',
-                        '3xx@tarantool.org',
-                        'mail.body')
+                       '3xx@tarantool.org',
+                       'mail.body')
     test:is(r.reason, 'RCPT failed: 354 (SMTP error)', 'Errors 3xx')
 
     r = client:request(addr, 'sender@tarantool.org',
-                        '4xx@tarantool.org',
-                        'mail.body')
+                       '4xx@tarantool.org',
+                       'mail.body')
     test:is(r.reason, 'RCPT failed: 421 (SMTP error)', 'service unavailable')
 
     r = client:request(addr, 'sender@tarantool.org',
-                        '5xx@tarantool.org',
-                        'mail.body')
+                       '5xx@tarantool.org',
+                       'mail.body')
     test:is(r.reason, 'RCPT failed: 510 (SMTP error)', 'unexisting recipient')
 
     r = client:request(addr, 'sender@tarantool.org',
-                        '10xx@tarantool.org',
-                        'mail.body')
+                       '10xx@tarantool.org',
+                       'mail.body')
     test:is(r.reason, 'response reading failed (SMTP error)', 'unexisting recipient')
 
 end)
```

Reply:
Thank you!

Reply:
Seems resolved, so I'll close the discussion.